docs/user-guide/slicing.ipynb

```diff
@@ -153,23 +160,111 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d['x', 0] += 1.0\n",
-    "d['x', 1] += 2.0"
+    "a['x', 0:1].coords['x'] *= 2  # ok, modifies only coord value \"private\" to this x-slice\n",
+    "try:\n",
+    "    a['x', 0:1].coords['y'] *= 2  # not ok, would modify coord value \"shared\" by all x-slices\n",
+    "except sc.VariableError as e:\n",
+    "    print(f'\\'y\\' is shared with other \\'x\\'-slices and should not be modified by the slice, so we get an error:\\n{e}')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "By excluding scalar items from the slice view (see below for a visual representation), we prevent unintentional addition of multiple offsets to the same scalar.\n",
-    "\n",
-    "This is an important aspect and it is worthwhile to take some time and think through the mechanism.\n",
-    "Consider the following example, contrasting slicing with and without range:\n",
+    "In practice, a much more dangerous issue this mechanism protects from is unintentional changes to masks.\n",
+    "Consider"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "val = a['x', 1]['y', 0].copy()\n",
+    "val"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we now assign this scalar `val` to a slice at `x=0`, using `=` we need to update the mask.\n",
+    "However, the mask in this example depends only on `y` so it also applies to the slices `x=1` and `x=2`.\n",
+    "If we would allow updating the mask, the following would *unmask data for all* `x`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    a['x', 0] = val\n",
```

Comment:
Just tried on `dev` branch, I don't think this actually throws at all.

Reply:
Must have broken it again in some subsequent refactor. Good catch!

Reply:
Ok, fixed. It actually worked, I had just switched x and y in the array, without switching also in this example.